.github/workflows/ci.yaml

```diff
@@ -0,0 +1,86 @@
+name: CI Pipeline
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code from the repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Install Clean dependencies
+        working-directory: ./angular
```

Comment:
Can you set default working directory instead of repeating it for lots of steps?

Reply:
Done